src/services/inference/module_js.ml

```diff
@@ -157,13 +157,18 @@ module ReversePackageHeap = SharedMem.WithCache (StringKey) (struct
     let prefix = Prefix.make()
   end)
 
-let get_key key tokens = Ast.(
+let get_key' key tokens = Ast.(
   match SMap.get (spf "\"%s\"" key) tokens with
```

Comment:
Oh but you should probably get rid of this `spf` since you're doing it in `get_key`. The double quoting is probably not helping the tests.

Or only do the quoting in here. Your call!